src/components/data-exploration/generic-gene-table/CreateCellSetModal.jsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import {
+  Modal, Form, Button, Select, InputNumber, Row, Alert, Col,
+} from 'antd';
+import PropTypes from 'prop-types';
+
+const formItemStyle = { margin: '0.375rem 0' };
+
+const CreateCellSetModal = (props) => {
+  const { selectedGenes, onCancel } = props;
+  const [form] = Form.useForm();
+  const intialFormValues = selectedGenes.map((gene) => ({ selectedGenes: gene }));
```

Comment:
The `CreateCellSetModal` will need to be tied up to the redux store later on anyway to implement the calls to create cellset actions, so there is no point passing `selectedGenes` as props and avoiding Redux right now. In this case, you can have `  const selectedGenes = useSelector((state) => state.genes.selected);` and remove variable `intialFormValues`, just use `selectedGenes`

Reply:
Also, why are you representing `initialFormValues` as an array of objects like this:
`[ {selectedGenes: 'CD3D'},  {selectedGenes: 'PPBP'},  {selectedGenes: 'Chil3'}]`? Why not representing it like this:
`['CD3D', 'PPBP', 'Chil3']`?

Reply:
because the form needs to know where these fields should be assigned to - `selectedGenes` is the first text field